build.gradle

```diff
@@ -24,10 +24,10 @@ sourceCompatibility = '1.8'
 group = 'to.wetransform'
 
 project.ext {
-  haleVersion = '3.3.1'
+  haleVersion = '3.4.0-SNAPSHOT'
```

Comment:
The way I managed it before was that only the `devel` branch uses the current snapshot of hale and the `master` uses the current release.

That's the basic difference between those branches. `master` allows to pull new commands and changes to commands in, based on the hale release. `devel` may use functionality that is not available in the release.